spec/factories/photos.rb

```diff
@@ -0,0 +1,7 @@
+FactoryGirl.define do
+  factory :photo do
+    title "test"
+    image  Rack::Test::UploadedFile.new(Rails.root + 'spec/assets_specs/photos/test_photo1.jpg', 'image/jpg')
```

Comment:
Line is too long. [109/80]<br>Unnecessary spacing detected.<br>Put one space between the method name and the first argument.